Modules/WikiPages.psm1

```diff
@@ -0,0 +1,117 @@
+<#
+.SYNOPSIS
+
+Write-DscResourceWikiSite generates wiki pages that can be uploaded to GitHub to use as 
+public documentation for a module.
+
+.DESCRIPTION
+
+The Write-DscResourceWikiSite cmdlet will review all of the MOF based resources
+in a specified module directory and will output the Markdown files to the specified directory. 
+These help files include details on the property types for each resource, as well as a text 
+description and examples where they exist.
+
+.PARAMETER OutputPath
+
+Where should the files be saved to
+
+.PARAMETER ModulePath
+
+The path to the root of the DSC resource module (where the PSD1 file is found, not the folder for
+and individual DSC resource)
+
+.EXAMPLE 
+
+This example shows how to generate help for a specific module
+
+    Write-DscResourceWikiSite -ModulePath C:\repos\SharePointdsc -OutputPath C:\repos\SharePointDsc\en-US
+
+#>
+function Write-DscResourceWikiSite {
+    param
+    (
+        [parameter(Mandatory = $true)] 
+        [System.String] 
+        $OutputPath,
+
+        [parameter(Mandatory = $true)] 
+        [System.String] 
+        $ModulePath
+    )
+
+    Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "MofHelper.psm1")
+    
+    $mofSearchPath = (Join-Path -Path $ModulePath -ChildPath "\**\*.schema.mof")
+    $mofSchemas = Get-ChildItem -Path $mofSearchPath -Recurse 
+    $mofSchemas | ForEach-Object {
+        $mofFileObject = $_
+        $result = (Get-MofSchemaObject $_.FullName) | Where-Object { 
+            ($_.ClassName -eq $mofFileObject.Name.Replace(".schema.mof", "")) `
+                -and ($null -ne $_.FriendlyName)  
+        }
+
+        $descriptionPath = Join-Path -Path $_.DirectoryName -ChildPath "readme.md"
+        if (Test-Path -Path $descriptionPath)
+        {
+            Write-Output "Generating wiki page for $($result.FriendlyName)"
```

Comment:
This probably should be a write-progress or write-verbose (in appveyor write-verbose with the -verbose flag if you want it to show up)